benchmarks/babelstream/src/AlpakaStream.h

```diff
@@ -42,8 +42,8 @@ struct AlpakaStream : Stream<T>
     using DevHost = alpaka::Dev<PlatformHost>;
     using PlatformAcc = alpaka::Platform<Acc>;
     using DevAcc = alpaka::Dev<Acc>;
-    using BufHost = alpaka::Buf<alpaka::DevCpu, T, Dim, Idx>;
-    using BufAcc = alpaka::Buf<Acc, T, Dim, Idx>;
+    using BufHost = alpaka::Buf<alpaka::DevCpu, T, Dim, Idx, alpaka::MemVisibilityTypeList<alpaka::DevCpu>>;
```

Comment:
Defining the buffer type looks ugly now. @chillenzer suggested to use a default template parameter: `typename TMeVisibility = alpaka::MemVisibilityTypeList<TDev>`. This is possible but this also means the default is, that the memory is only visible from the plattform of given device type.

If we want to use for example manage memory, we still need to define the visibility manually. Also it could confuse, if the user defines the an alpaka buffer with default visibility as member variable and tries to use `alpakaMemMapped` for allocating the memory.